src/EmojiTranslatorContainer.js

```diff
@@ -2,6 +2,14 @@ import React, { Component } from 'react';
 import Translator from './components/Translator';
 import Stories from './components/Stories';
 
+const emojiMap = {
```

Comment:
You might consider moving this to a different file and importing it in, in case you need this emoji list in multiple different places. Probably not needed right now